package.json

```diff
@@ -37,6 +37,7 @@
     "prettier": "prettier --config .prettierrc.json --write '**/**/**/**/*.js' '**/**/**/*.js' '**/**/*.js' '**/*.js' '*.js'",
     "prettier:text": "prettier 'README.md' 'doc/*.md' 'doc/**/*.md' --no-config --tab-width 4 --print-width 120 --write --prose-wrap always",
     "test": "nyc --reporter=text mocha --recursive 'test/**/*.js' --reporter spec --timeout 5000 --ui bdd --exit",
+    "test:debug": "mocha --recursive 'test/**/*.js' --reporter spec --inspect-brk --timeout 30000 --ui bdd --exit" ,
```

Comment:
@AlvaroVega this would help in debugging, if you wanna keep it, I will document it

Reply:
Good idea! Doc is wellcome.

Reply:
> Good idea! Doc is welcome.

fixed by https://github.com/telefonicaid/iotagent-node-lib/pull/952/commits/fef8475ea80e8e1ea2ab2ea4ddd99630f42e9368